packages/client/src/DirectoryClient.ts

```diff
@@ -55,7 +72,6 @@ export class DirectoryClient {
                 legalOfficers.push(new LegalOfficerClass({
                     legalOfficer,
                     axiosFactory: this.axiosFactory,
-                    token: this.token,
```

Comment:
`LegalOfficerClass` still needs the token to fetch config.

Reply:
Of course ! Many thanks for pointing this.